sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerLanguage.java

```diff
@@ -12,6 +12,13 @@
  * <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/language-support?tabs=v2-1">here</a>.
  */
 public final class FormRecognizerLanguage extends ExpandableStringEnum<FormRecognizerLanguage> {
+
+    /**
+     * Creates a FormRecognizerLanguage object.
+     */
+    public FormRecognizerLanguage() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.